### ISSUE TITLE
config.js always overwritten with docker images

### ISSUE THREAD

**Issue by nicolass67** [xwiki-labs/cryptpad#349]
Hello, 
I recently updated my docker image of cryptpad building the docker file. I saw a little problem with the config.js. 
To reproduce : 

- build the image 
- run the container with the volumes mounted as seen in docs, especially the customize volume
- change something in the <yourpathtovolume>/customize/config.js (for example add a comment) 
- stop / rm / start the container the same way 
- your comment or config instructions have disappearred in customize/config.js 

Can it be related to : https://github.com/xwiki-labs/cryptpad/pull/344 ? 

Thanks in advance.

**Comment by ansuz** [comment]
@mcrosson can you comment on this?

**Comment by mcrosson** [comment]
This is accurate. I ran into it yesterday and hadn't had a chance to follow up with a ticket myself.

Definitely related to #344

**Comment by nicolass67** [comment]
Hello, 

I just tried this workaround in the container-start.sh file : 
Put this : 

```
# Copying default config
[ ! -r customize/config.js ] && echo "Creating config.js" && cp config.example.js customize/config.js
```

instead of using 
`[ ! -h customize/config.js ] ... `

I rebuilt the docker images, and now the customize/config.js ins't overwritten. 

But I come back to my problem in : https://github.com/xwiki-labs/cryptpad/issues/339 
I'll comment it separately to distinguish the 2 issues. 

Regards,

**Comment by ineiti** [comment]
Fixed in #351

**Comment by nicolass67** [comment]
Thanks for the fix. 

Can someone re-open the #339 because now the problem reappears ... thanks (I know it's not the purpose of this issue).
